ndcube/asdf/resources/schemas/NDCube-0.1.0.yaml

```diff
@@ -0,0 +1,21 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/NDCube-0.1.0"
+
+title: |
+  Represents the ndcube NDCube object
+
+description:
+  Represents the ndcube NDCube object
+
+type: object
+properties:
+  data:
+    tag: "tag:stsci.edu:asdf/core/ndarray-1.0.0"
+  wcs:
+    $ref: "http://stsci.edu/schemas/gwcs/wcs-1.1.0"
```

Comment:
```suggestion
    $ref: "http://stsci.edu/schemas/gwcs/wcs-1.*"
```

Reply:
I think we can't use wildcard for `$ref`, I have used `anyOf` to incorporate different versions, is this fine?

Reply:
Good catch. Yes, the wildcard will only work if this is a tag. Any reason not to use the tag instead of a `$ref`? I'll update the suggestion with what I think should work.

Reply:
I was experimenting and went with `$ref`, after reading the docs, I think it would be better to use a tag here. I believe `$ref` would be useful when we reuse the local schemas.